test/studies/lcals/LongDouble.chpl

```diff
@@ -0,0 +1,39 @@
+/* A module to provide a "long double" type to Chapel that matches the
+   underlying C compiler's "long double" type.
+ */
+
+require "longdouble.c", "longdouble.h";
+
+extern type longdouble;
+
+extern proc longdoubleToDouble(d: longdouble): real(64);
+
+proc _cast(type t, ld: longdouble) where t == real {
+  return longdoubleToDouble(ld);
```

Comment:
I don't really understand why you need `longdoubleToDouble()` Will a primitive cast not work here?